openproblems/tasks/dimensionality_reduction/methods/densmap.py

```diff
@@ -30,6 +30,7 @@ def _densmap(adata, obsm=None):
 @_densmap_method(method_name="densMAP (logCPM, 1kHVG)")
 def densmap_logCPM_1kHVG(adata, test: bool = False):
     adata = log_cpm_hvg(adata)
+    adata = adata[:, adata.var["highly_variable"]].copy()
```

Comment:
do we need to call `log_cpm_hvg` differently then? Or okay to leave it like this as it still annotates HVGs? @lazappi ?

Reply:
I changed it (added this line) in every instance where it is called, meaning overall functionality is unchanged -- I have just moved the subsetting out of the normalizer and into the method.

Reply:
yeah, my question was on naming of the function above... but i guess okay to not rename